server/service/src/requisition/request_requisition/generate.rs

```diff
@@ -74,7 +74,7 @@ pub fn generate_requisition_lines(
                 requested_quantity: 0.0,
                 approved_quantity: 0.0,
                 approval_comment: None,
-                initial_stock_on_hand_units: 0.0,
+                initial_stock_on_hand_units: available_stock_on_hand,
```

Comment:
Initial stock on hand is the stock on hand at the start of the period.. so I feel this isn't right either 🤔